### PR TITLE
Configurator bootstrap + ColorEditor

### PR DESCRIPTION
**Pull request by veej**
https://github.com/buildo/bento-design-system/assets/925635/414628a4-23d7-4111-a2ea-c20f0208a642

### CODE REVIEW

File: packages/configuration-builder/src/ColorEditor/Palette.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                  display="flex"
+                  width={24}
+                  height={24}
+                  background="backgroundLightScrim"
+                  borderRadius={8}
+                  alignItems="center"
+                  justifyContent="center"
+                  margin="negative8"
+                >
+                  <IconLockSimple size={16} color="primary" />
```
Comment (gabro) [inline packages/configuration-builder/src/ColorEditor/Palette.tsx:81]: This is not very accessible, can we add an aria-label?

File: packages/configuration-builder/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  const [value, setValue] = useState<ColorConfig>({
+    keyColor: "#1A212B" as HexColor,
+    hue: 215,
+    saturation: 25,
+    lightnessInterpolation: "Linear",
+    keyColorLocked: true,
+  });
+
+  return (
+    <ColorEditor name={unsafeLocalizedString("Brand color")} value={value} onChange={setValue} />
```
[the remark this answers is not in the document]
Comment (veej) [reply packages/configuration-builder/src/App.tsx:16]: It's temporary, just for testing purposes. I didn't create a localization key on purpose.

File: packages/configuration-builder/src/ColorEditor/Palette.tsx (hunk 1 above)
Comment (veej) [reply packages/configuration-builder/src/ColorEditor/Palette.tsx:81]: makes sense 👍

**Review by federico-ercoles — APPROVED** [review]
LGTM 👍🏻

**Review by gabro — APPROVED** [review]
One minor thing, but approved nonetheless

File: packages/configuration-builder/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    /* Linting */
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true
+  },
+  "include": ["src"],
+  "references": [{ "path": "./tsconfig.node.json" }]
+}
```
[the remark this answers is not in the document]
Comment (veej) [reply packages/configuration-builder/tsconfig.json:25]: This is all auto-generated by `pnpm create vite --template react-ts`, so not created manually anyway. I'll have a look at the options in @tsconfig/vite-react, but I think we're good with the default config for now.

[the remark this answers is not in the document]
Comment (gabro) [reply packages/configuration-builder/tsconfig.json:25]: my concern is about maintainability: we rarely update tsconfigs and sometimes we miss out on new features/options. Delegating to an external library makes it easier

[the remark this answers is not in the document]
Comment (veej) [reply packages/configuration-builder/tsconfig.json:25]: makes sense 👍

**Pull request merged**